build.gradle

```diff
@@ -19,25 +19,21 @@ dependencies {
     testImplementation 'com.browserstack:browserstack-local-java:1.0.6'
     testImplementation 'org.apache.commons:commons-lang3:3.11'
     testImplementation 'org.slf4j:slf4j-api:1.7.30'
-    testImplementation 'ch.qos.logback:logback-classic:1.2.3'
     testImplementation 'org.testng:testng:7.4.0'
     testImplementation 'io.cucumber:cucumber-java:6.10.4'
     testImplementation 'io.cucumber:cucumber-core:6.10.4'
     testImplementation 'io.cucumber:cucumber-picocontainer:6.10.4'
     testImplementation 'io.cucumber:cucumber-testng:6.10.4'
     testImplementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.12.2'
     testImplementation 'com.fasterxml.jackson.core:jackson-databind:2.12.2'
+    testImplementation 'com.browserstack:webdriver-framework-testng:0.0.1'
 }
 
-group = 'com.browserstack'
-version = '0.0.1-SNAPSHOT'
-description = 'browserstack-examples-cucumber-testng'
-java.sourceCompatibility = JavaVersion.VERSION_1_8
```

Comment:
Reason to remove?

Reply:
we are picking up from remote URL right?

I added - "testImplementation 'com.browserstack:webdriver-framework-testng:0.0.1'"

I believe the above code picks up the lib locally. is that needed?